homeassistant/components/light/wemo.py

```diff
@@ -105,6 +105,7 @@ def turn_on(self, **kwargs):
         elif ATTR_RGB_COLOR in kwargs:
             xycolor = color_util.color_RGB_to_xy(
                 *(int(val) for val in kwargs[ATTR_RGB_COLOR]))
+            self.device.turn_on(level=xycolor[2])
```

Comment:
Looking at how hue and wink are simply updating values in a dict which is passed down, I should probably at some point change the wemo light support to do the same. `set_color` and `set_temperature` right now end up just queueing values for the next `turn_on` call, under the covers all these zigbee lights use the same `SetDeviceStatus` operation to updates all values at once.